ydb/docs/en/core/integrations/orm/linq2db.md

```diff
@@ -0,0 +1,503 @@
+| `DateTime`                    | `DateTime`                                | `Datetime`         | Second precision.                                                     |
+| `DateTime` / `DateTimeOffset` | `DateTime2`                               | `Timestamp`        | Microseconds.                                                         |
+| `TimeSpan`                    | `Interval`                                | `Interval`         | Duration.                                                             |
+| `decimal`                     | `Decimal`                                 | `Decimal(p,s)`     | Defaults to `Decimal(22,9)`. Use `Decimal(p,s)` when feature enabled. |
+| `float`                       | `Single`                                  | `Float`            | —                                                                     |
+| `double`                      | `Double`                                  | `Double`           | —                                                                     |
+| `sbyte` / `byte`              | `SByte` / `Byte`                          | `Int8` / `Uint8`   | —                                                                     |
+| `short` / `ushort`            | `Int16` / `UInt16`                        | `Int16` / `Uint16` | —                                                                     |
+| `int` / `uint`                | `Int32` / `UInt32`                        | `Int32` / `Uint32` | —                                                                     |
+| `long` / `ulong`              | `Int64` / `UInt64`                        | `Int64` / `Uint64` | —                                                                     |
+| `string`                      | `Json`                                    | `Json`             | Text JSON.                                                            |
+| `byte[]`                      | `BinaryJson`                              | `JsonDocument`     | Binary JSON.                                                          |
+| `DateOnly` / `DateTime`       | `Date`                                    | `Date32`           | **New type.** Wider date range. Use `DbType = "Date32"`.              |
+| `DateTime`                    | `DateTime`                                | `Datetime64`       | **New type.** Second precision, wider range. `DbType = "Datetime64"`. |
+| `DateTime` / `DateTimeOffset` | `DateTime2`                               | `Timestamp64`      | **New type.** Microseconds, wider range. `DbType = "Timestamp64"`.    |
+| `TimeSpan`                    | `Interval`                                | `Interval64`       | **New type.** Wider interval range. `DbType = "Interval64"`.          |
+
+> You can set exact `Precision`/`Scale` with attributes:  
+    >   `[Column(DataType = DataType.Decimal, Precision = 22, Scale = 9)]`  
+    >   or globally via `YdbOptions(UseParametrizedDecimal: true)`.
```

Comment:
1. Make a full example out of this
2. In the "globally via" case it's unclear how to specify the Precision and Scale